src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+import duke.exception.DukeException;
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.Ui;
+import duke.task.Task;
+
+/**
+ * Represents a command to add a deadline task to the task list.
+ */
+public class DeleteCommand extends Command {
+
+    private int zeroItem;
```

Comment:
It seems like this variable is representing an index. Perhaps you can consider giving a clearer name, such as "indexToDelete"?